core/sr-api-macros/tests/ui/declaring_own_block_with_different_name.stderr

```diff
@@ -4,10 +4,10 @@ error: `Block: BlockT` generic parameter will be added automatically by the `dec
 5 |     pub trait Api<B: BlockT> {
   |                      ^^^^^^
 
-warning: unused import: `runtime_primitives::traits::Block as BlockT`
+error: unused import: `runtime_primitives::traits::Block as BlockT`
```

Comment:
If there is an `error` instead of a `warning`, then  `-D warnings` will not stumble here.  Thus, note does not make sense.